test/fixtures/dep-go-1-10/Gopkg.toml

```diff
@@ -0,0 +1,25 @@
+[metadata.heroku]
+  root-package = "github.com/heroku/fixture"
+  go-version = "go1.10.2"
```

Comment:
change this to `go1.10` and it will expand to the most recent version of go1.10.X

Reply:
Fixed with `--amend`

Reply:
I just realized we couldn't use `go1.10` here since the test is asserting a folder having the go version there: https://github.com/heroku/heroku-buildpack-go/pull/245/commits/f4a7f0a2bd7d883ecfae37fc0942450e21c34551#diff-a6e781883513014286df21a2e0e8aaa2R973. It has to be a specific version.